### PR TITLE
report-list: add container

### PR DESCRIPTION
**Pull request by alalbuquerque**
### Contexto
O container ReportList deverá ser usado na página de relatórios. Ele deverar receber um objeto chamado reports e irá renderizar a lista de relatórios vindo do tractor. 

### Exemplo
screenshot from 2018-06-04 13-35-08
screenshot from 2018-06-20 14-29-30

### Descrição técnica
```jsx
<ReportList
  reports={reports}
/>
```
### Requisitos e Critérios de Aceite
- [ ] Renderizar o componente de ReportCard em listagem
- [ ] Paginação funcional 
- [ ] Controle de collapse nos itens da listagem
- [ ] Contador de itens de relatório
- [ ] Tratar labels e descrições de status dos relatórios
- [ ] Fazer tradução dos dados 
- [ ] Itens com ações diferenciados por tipos de status
- [ ] Interação com o filtro trazendo item por datas, status e tipo de relatórios

### CODE REVIEW

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 728d01dbb3289514587d59308a94fc4f7e2170d5
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 8d53f2e491a3a26802fb20e656c8d3404308da6c
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 211b77bdf8c4685a1edf5aa62acc994ce688c468
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 10dfe61b1a87e13aee53af6cdbc60851daea81db
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 24e2ae5ccb0c626b47f5c61e519154f47427d067
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 453268a781636501afdb8f05b2d2818597bab0ca
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: f80a2b3325ea0f276be389d6a0fae801c9863c8b
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: e19f0782107d51eaac607049b6a4d2bae55fb47c
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 88f2061f36d2dbfc1f9adb82662d23d59024f1de
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ee9ca5407e9416f3b82ce0e9bc7f2c92afe85e40
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 1bcbd4acc92e396dd25ccadad58f6cc1d45d9714
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0ad23da483ba2650542b92c34c82d8c6ea13474a
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 9d223292d1f7278e6c6c5e27df43bf48c46e400b
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: de27e582c14d296de9c3116048a30befc753dc08
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ab1367e4f93074c3e8160c67223852fdf1b68a78
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 728d01dbb3289514587d59308a94fc4f7e2170d5
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 8d53f2e491a3a26802fb20e656c8d3404308da6c
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 211b77bdf8c4685a1edf5aa62acc994ce688c468
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 10dfe61b1a87e13aee53af6cdbc60851daea81db
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 24e2ae5ccb0c626b47f5c61e519154f47427d067
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 453268a781636501afdb8f05b2d2818597bab0ca
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: f80a2b3325ea0f276be389d6a0fae801c9863c8b
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: e19f0782107d51eaac607049b6a4d2bae55fb47c
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 88f2061f36d2dbfc1f9adb82662d23d59024f1de
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ee9ca5407e9416f3b82ce0e9bc7f2c92afe85e40
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change
  * Good titles doesn't exceed 50 characters. To add more details use the body!

 * Commit: 1bcbd4acc92e396dd25ccadad58f6cc1d45d9714
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0ad23da483ba2650542b92c34c82d8c6ea13474a
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 9d223292d1f7278e6c6c5e27df43bf48c46e400b
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: de27e582c14d296de9c3116048a30befc753dc08
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ab1367e4f93074c3e8160c67223852fdf1b68a78
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: a530a4c500404e0d434b0ff947f5bdcd7e715e1f
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0f4bbd7e5bb53accf42dc94a9fb897ebf25adcf6
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: c974740559d9ac4dc222d96a17ba0bc756e10fd0
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 5863abd389ac8add7d9691195ad3047094f16a95
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: d99254a38de93a85ab8108c8d35a53dcb139317e
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 01237e1a621f10edf6806c56c7505fe44a2af99f
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ba1c387f73b1251b5eef3ebfa5dc334ad2853e93
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7b74818c0fa3013358f7b103a4c8160f92112ff0
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: cf64bf16f5795f1a6a28828cc97cd64612a16edd
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6e4d51a97ce8c48be1ea535b320ab690bf4b32d3
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: a530a4c500404e0d434b0ff947f5bdcd7e715e1f
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0f4bbd7e5bb53accf42dc94a9fb897ebf25adcf6
  * Good titles doesn't exceed 50 characters. To add more details use the body!
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: c974740559d9ac4dc222d96a17ba0bc756e10fd0
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 5863abd389ac8add7d9691195ad3047094f16a95
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: d99254a38de93a85ab8108c8d35a53dcb139317e
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 01237e1a621f10edf6806c56c7505fe44a2af99f
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: ba1c387f73b1251b5eef3ebfa5dc334ad2853e93
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7b74818c0fa3013358f7b103a4c8160f92112ff0
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: cf64bf16f5795f1a6a28828cc97cd64612a16edd
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6e4d51a97ce8c48be1ea535b320ab690bf4b32d3
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by uselessdev** [comment]
@alalbuquerque esse PR é WIP?

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: ae3cc61ddfa8ba64c52c6672c4e0beffd51649b5
  * Always use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by alalbux** [comment]
Aguardando aprovação desse cara https://github.com/pagarme/pilot/pull/660 pra fazer um rebase e passar no CI

**Comment by rsmelo** [comment]
este PR depende do merge do PR https://github.com/pagarme/pilot/pull/852

File: packages/pilot/stories/containers/ReportList/style.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    display: flex;
+}
+
+.cardComponent > * {
+  margin-left: 17px;
+}
+  
+.cardComponent > *:first-child {
+  margin-left: unset;
+}
```
Comment (lucasjs) [inline packages/pilot/stories/containers/ReportList/style.css]: Falta a linha em branco.

File: packages/pilot/stories/containers/ReportList/style.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.cardComponent {
+    display: flex;
```
Comment (lucasjs) [inline packages/pilot/stories/containers/ReportList/style.css]: Indentação incorreta.

File: packages/pilot/stories/containers/ReportList/style.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+}
+
+.cardComponent > * {
+  margin-left: 17px;
+}
+
```
Comment (lucasjs) [inline packages/pilot/stories/containers/ReportList/style.css]: Espaço desnecessário.

File: packages/pilot/src/containers/ReportList/style.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    display: flex;
+}
+
+.cardComponent > * {
+  margin-left: 17px;
+}
+  
+.cardComponent > *:first-child {
+  margin-left: unset;
+}
```
Comment (lucasjs) [inline packages/pilot/src/containers/ReportList/style.css]: Pular linha.

File: packages/pilot/src/containers/ReportList/style.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.cardComponent {
+    display: flex;
+}
+
+.cardComponent > * {
+  margin-left: 17px;
+}
+
```
Comment (lucasjs) [inline packages/pilot/src/containers/ReportList/style.css]: Espaço desnecessário.

File: packages/pilot/src/containers/ReportList/style.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+.cardComponent {
+    display: flex;
```
Comment (lucasjs) [inline packages/pilot/src/containers/ReportList/style.css]: 4 espaços ao invés de 2.

File: packages/pilot/src/containers/ReportList/index.js [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+    )
+  }
+
+  return (
+    <Fragment>
+      <Button
+        fill="outline"
+        icon={<TrashIcon width={16} height={16} />}
+        onClick={() => null}
+        size="default"
```
Comment (lucasjs) [inline packages/pilot/src/containers/ReportList/index.js]: O `Button` já vem como default nas props.

**Comment by uselessdev** [comment]
Precisa fazer um `squash` nesses commits

**Comment by uselessdev** [comment]
Tem alguns erros no terminal relacionado com o que o @lucasjs comentou sobre o css. Fora isso acho que ta ok.

**Comment by rsmelo** [comment]
Este PR está parado a mais de 1 ano, por isso estou fechando ele

**Comment by LeoNero** [comment]
:'(

**Pull request closed without merging**